portfolio/src/main/webapp/aboutme.html

```diff
@@ -1 +1,43 @@
+<!DOCTYPE html>
```

Comment:
A lot of this file seems the same as about.html.
Is that intentional to have the translate feature in both of them?